CHANGELOG.md

```diff
@@ -3,6 +3,22 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.8.4](https://github.com/okta/odyssey/compare/v0.8.3...v0.8.4) (2021-11-18)
+
+### Features
+
+- expose hidden choices reference ([a64597a](https://github.com/okta/odyssey/commit/a64597a57f41bbbb1053d9e1a7254148d9318875))
+- **odyssey-react:** Text component refactor in Banner ([#1160](https://github.com/okta/odyssey/issues/1160)) ([8795f01](https://github.com/okta/odyssey/commit/8795f0172e34fd1ebdf86f3f854ef8539a7ada70))
+- support async select options ([0bc1986](https://github.com/okta/odyssey/commit/0bc1986e01d6f843cd589c56d4cdda27b629750f))
+
+### Bug Fixes
+
+- controlled select case by using to ref.current ([b61b2bd](https://github.com/okta/odyssey/commit/b61b2bd0c448a3b01fc85a7c6233a9b69a4409a4))
+- lint ([617b736](https://github.com/okta/odyssey/commit/617b736038b508b8747c78099cca2b7202cc0846))
+- remove added dep / prop ([a349727](https://github.com/okta/odyssey/commit/a3497270d0369fa3bd0b1d8571fef698d28c27c4))
+- type issue need cast to force union start type when using callback ([2f8f482](https://github.com/okta/odyssey/commit/2f8f482bb42737dca5f32e97384fe2c83f6f7cbb))
+- updates per review feedback ([10cf21e](https://github.com/okta/odyssey/commit/10cf21ef22c13e3affb1cec57ba3bb8e8b218256))
```

Comment:
its worth noting here that these bugfixes were never affecting previous releases of the project - only previous commits on topic branches so the documentation here begins to become less useful unless folks are crafting their conventional commit messages with the correct context (release docs) in mind.

One way to enforce better outcomes is to squash merge + fast forward and group topic branches down into single commits.

Reply:
I'm tempted to remove some of these messages entirely here - and from the `odyssey-react/CHANGELOG` below as they do not make 100% sense for the intended audience